dbt_subprojects/daily_spellbook/models/_projects/gmx/tokens/arbitrum/gmx_v2_arbitrum_CollateralTokensData.sql

```diff
@@ -0,0 +1,17 @@
+{{
+  config(
+    schema = 'gmx_v2_arbitrum',
+    alias = 'collateral_tokens_data',    
+    materialized = 'view',
+    post_hook='{{ expose_spells(\'["arbitrum"]\',
+                                "project",
+                                "gmx",
+                                \'["ai_data_master","gmx-io"]\') }}'
```

Comment:
```suggestion
    materialized = 'view'
```

Reply:
lets remove post hook to display on data explorer from these upstream dependency models, as they are mostly intended to help final spells downstream. we are trying to keep the data explorer a bit more tidy.

Reply:
Done